goas/cl/proof_statements/src/lib.rs

```diff
@@ -1 +1,2 @@
 pub mod input;
+pub mod zone_funds;
```

Comment:
The purpose of the `cl/` sub-crate is to model the stuff that is CL specific and zone/ should have all the user space stuff.

Ideally, zone_funds is a user space concept.

We can continue like this for now, but we'll want to decouple this in the future and move the zone death constraints out of CL